### PR TITLE
Update UI

### PR DESCRIPTION
**Pull request by Wgis**
New look for UI 
https://i.imgur.com/X9JAgEr.png (AutoSize by config pocket item count)

### CODE REVIEW

File: entities/weapons/pocket/cl_menu.lua [hunk 1]
```diff
@@ -32,14 +32,27 @@ function DarkRP.openPocketMenu()
     if not wep:IsValid() or wep:GetClass() ~= "pocket" then return end
     if not pocket then pocket = {} return end
     if table.Count(pocket) == 0 then return end
-    frame = vgui.Create("DFrame")
-
+    
+	local count = GAMEMODE.Config.pocketitems or GM.Config.pocketitems
```
Comment (FPtje) [inline entities/weapons/pocket/cl_menu.lua]: You're using tabs for indentation, even though the rest of the file is using spaces

File: entities/weapons/pocket/cl_menu.lua [hunk 2]
```diff
@@ -54,16 +67,18 @@ function reload()
 
     local itemCount = table.Count(pocket)
 
-    frame:SetSize(itemCount * 64, 90)
-    frame:Center()
-
+--    frame:SetSize(itemCount * 64, 90)
```
Comment (FPtje) [inline entities/weapons/pocket/cl_menu.lua]: Forgot to remove?

File: entities/weapons/pocket/cl_menu.lua [hunk 3]
```diff
@@ -54,16 +67,18 @@ function reload()
...
-    for k, v in pairs(pocket) do
 
-        local icon = vgui.Create("SpawnIcon", frame)
-        icon:SetPos(i * 64, 25)
+    for k, v in pairs(pocket) do
+		local ListItem = frame.List:Add( "DPanel" ) 
+		ListItem:SetSize( 64, 64 ) 
+	
+        local icon = vgui.Create("SpawnIcon", ListItem)
+    --    icon:SetPos(i * 64, 25)
```
Comment (FPtje) [inline entities/weapons/pocket/cl_menu.lua]: Forgot to remove?

**Comment by Wgis** [comment]
Sorry, but i'm bad in git....

**Pull request closed without merging**